rules/autoconsent/pubtech.json

```diff
@@ -1,30 +1,30 @@
 {
-    "name": "pubtech",
-    "prehideSelectors": ["#pubtech-cmp"],
-    "detectCmp": [
-      { 
-        "exists": "#pubtech-cmp" 
-      }
-    ],
-    "detectPopup": [
-      { 
-        "visible": "#pubtech-cmp" 
-      }
-    ],
-    "optIn": [
-      { 
-        "click": "#pubtech-cmp #pt-accept-all" 
-      }
-    ],
-    "optOut": [
-      {
-        "click": "#pubtech-cmp #pt-close:first-child" 
-      }
-    ],
-    "test": [
-      {
-        "eval": "document.cookie.includes('euconsent-v2=CPeDpcAPeDpcAFgADAENCaCgAAAAAAAAABCYAAAAAAAA.YAAAAAAAAAAA') && !document.cookie.includes('IDE')"
-      }
-    ]
-  }
-  
\ No newline at end of file
+  "name": "pubtech",
+  "prehideSelectors": ["#pubtech-cmp"],
+  "detectCmp": [
+    { 
+      "exists": "#pubtech-cmp" 
+    }
+  ],
+  "detectPopup": [
+    { 
+      "visible": "#pubtech-cmp #pt-accept-all, #pt-actions"
+    }
+  ],
+  "optIn": [
+    {
+      "click": "#pubtech-cmp #pt-actions button:nth-of-type(2)"
+    }
+  ],
+  "optOut": [
+
+    {
+      "click": "#pubtech-cmp #pt-close"
+    } 
+  ],
+  "test": [
+    {
+      "eval": "document.cookie.includes('euconsent-v2') && document.cookie.match(/.YAAAAAAAAAAA/) || document.cookie.match(/.aAAAAAAAAAAA/) || document.cookie.match(/.YAAACFgAAAAA/) "
```

Comment:
I think you're missing some parentheses here around the OR'ed parts

Reply:
Not sure I understand. I had a look at another json eval rule and parts with OR were not under parenthesis. Let me know if I misunderstood it.

Reply:
`A && B || C || D` and `A && (B || C || D)` are two different things, and I think you meant the latter here